resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/internal/CircuitBreakerMetrics.java

```diff
@@ -130,17 +134,23 @@ public Result onError(long duration, TimeUnit durationUnit) {
      */
     private Result checkIfThresholdsExceeded(Snapshot snapshot) {
         float failureRateInPercentage = getFailureRate(snapshot);
-        if (failureRateInPercentage == -1) {
-            return Result.BELOW_MINIMUM_CALLS_THRESHOLD;
+        float slowCallsInPercentage = getSlowCallRate(snapshot);
+
+        if (failureRateInPercentage == -1 || slowCallsInPercentage == -1) {
+            return Result.FAILURE_RATE_SLOW_CALL_RATE_BELOW_MINIMUM_CALLS_THRESHOLD;
```

Comment:
This PR renames the existing `Result` enum to `ComputeResult`. The reason for introducing one more indirection is because I'd like to know which threshold was breached - slow call rate or failure rate, so I can emit a separate event for each.

`FAILURE_RATE_SLOW_CALL_RATE_BELOW_MINIMUM_CALLS_THRESHOLD` tells me that both were breached. See my response to your question for more context: `Why do we need to check the outcome of a call at this point?`